tests/unit/test_pyramid_app/app_test.py

```diff
@@ -15,5 +17,26 @@ def test_index():
     assert app.index(sentinel.request) == {"Hello": "Pyramid!"}
 
 
-def test_status():
-    assert app.status(sentinel.request) == {"status": "okay"}
+def test_status(capture_message):
+    request = DummyRequest(path="/_status")
```

Comment:
There's no `pyramid_request` fixture in test-pyramid-app. We should add one to h-testkit so that all our apps get it. For now, just making dummy requests directly in the tests.